ui-admin/src/portal/siteContent/SiteContentEditor.tsx

```diff
@@ -31,6 +33,7 @@ const SiteContentEditor = (props: InitializedSiteContentViewProps) => {
     portalEnv, loadSiteContent, switchToVersion, createNewVersion, readOnly
   } = props
   const selectedLanguage = 'en'
+  const [activeTab, setActiveTab] = useState<string | null>('designer')
```

Comment:
outside the scope of this PR, but it might be nice to have these tabs be url params.